README.md

```diff
@@ -2,9 +2,147 @@
+
+If the proposal includes more than one alternative then there should be further discussion to try to
+single out the best alternative.
+
+These issues / PRs will be labeled as `proposal`s in the issue tracker.
+
+### Testing period
+
+If there are no objections to the proposal the new trait(s) will land behind the "unproven" Cargo
+feature and an issue about the new trait(s) will be created. If the proposal includes several
+alternatives and a single one couldn't be chosen as the best then each alternative will land behind
+a different Cargo feature, e.g. "alt1" or "alt2".
+
+The traits will undergo a testing period before they move into the set of proven traits. During
+this period users are encouraged to try to implement the unproven traits for their platforms and to
+build drivers on top of them. Problems implementing the trait(s) as well as successful
+implementations should be reported on the corresponding issue.
+
+To leave the unproven state at least *two* implementations of the trait(s) for different platforms
+(ideally, the two platforms should be from different vendors) and *one* generic driver built on top
+of the trait(s) must be demonstrated. If, instead, reports indicate that the proposed trait(s) can't
+be implemented for a certain platform then the trait(s) will be removed and we'll go back to the
+drawing board.
+
+Issues used to track unproven APIs will be labeled as `unproven-api`s in the issue tracker and they
+may also include the labels `needs-impl` and `needs-driver` to signal what's required for them to
+move to the set of proven traits.
+
+## Implementations
+
+These are (WIP) implementations of `embedded-hal` for various platforms. Feel free to send a PR
+adding yours to the list!
+
+### Linux
+
+- [`linux-embedded-hal`]. For the Raspberry Pi and other SBC that expose pins with embedded
+  functionality (SPI, I2C, etc.)
+
+[`linux-embedded-hal`]: https://crates.io/crates/linux-embedded-hal
+
+### Nordic
+
+- [`nrf51-hal`]. Check the [`microbit`] crate for examples that can be run on the [micro:bit]
+
+[`nrf51-hal`]: https://github.com/therealprof/nrf51-hal
+[`microbit`]: https://github.com/therealprof/microbit
+[micro:bit]: http://microbit.org/
+
+### NXP
+
+- [`lpc82x-hal`]
+
+[`lpc82x-hal`]: https://github.com/braun-robotics/rust-lpc82x-hal
+
+### ST Microelectronics
+
+- [`stm32f103xx-hal`]. Contains examples that can be run on the [Blue pill] and similar boards.
+
+[`stm32f103xx-hal`]: https://github.com/japaric/stm32f103xx
+[Blue pill]: wiki.stm32duino.com/index.php?title=Blue_Pill
+
+- [`stm32f30x-hal`]. Check the [`f3`] crate for examples that can be run on the STM32F3DISCOVERY.
+
+[`stm32f30x-hal`]: https://crates.io/crates/stm32f30x-hal
+[`f3`]: https://crates.io/crates/f3
+
+### Texas Instruments
+
+- [`tm4c123x-hal`]
+
+[`tm4c123x-hal`]: https://github.com/thejpster/tm4c123x-hal
+
+You may be able to find even more implementations by searching for the [`embedded-hal-impl`] keyword
```

Comment:
I think this paragraph is important, but it's easy to miss down here. A casual reader might think it's a note about TI. I suggest moving it up, making it the second paragraph under `## Implementations`.